website/contributing/release-troubleshooting.md

```diff
@@ -42,3 +42,50 @@ git pull <remote> <branch> --tags
 # Make sure you push any local tags you added/removed
 git push <remote> <branch> --follow-tags
 ```
+
+### The Tag Has Been Pushed but NPM Failed to Publish React Native
```

Comment:
first feedback: can you put these two sections as top ones, with the `Add a Local Commit` one first, then this, then the other two (I think it'd be better to have them in order of frequency, from the most likely one to the least)

Reply:
also: 
```suggestion
### The Git Tag Has Been Pushed to Remote but CI Failed to Release to NPM
```